index.js

```diff
@@ -4,6 +4,14 @@ var async = require('async')
 var MongoClient = require('mongodb').MongoClient
 
 function clean (db, done) {
+  cleanDb(db, null, done)
+}
+
+function cleanExclude (db, exclude, done) {
+  cleanDb(db, exclude, done)
+}
+
```

Comment:
You can just have an single clean function with an optional parameter, rather than two functions. Also, we might even have an `options` object for extensibility reasons, something like:

```js
clean(db, {
  exclude: ['dummy']
}, (err, db) => {})
```